### DISCUSSION TITLE
[FIXED] Equations in model vignettes not rendering in Chrome MacOS Ventura

### DISCUSSION THREAD

**Discussion by jamesmbaazam** [epiforecasts/EpiNow2#685] [General]
## Problem

I used to have an issue where the equations in the model vignettes would not render, especially where the LaTeX command `\mathcal` was used to typeset the equations. See the next image image for an example from the [`estimate_infections()` model vignette](https://epiforecasts.io/EpiNow2/articles/estimate_infections.html).

<img width="821" alt="Screenshot 2024-06-03 at 14 08 36" src="https://github.com/epiforecasts/EpiNow2/assets/43169674/98a03ba8-1c02-42df-89c8-f769f9478c2d">

## Platform

This only occurred in Chrome on my MacOS Ventura machine. It worked fine in Firefox. 

## Fix

Others have faced a similar issue. See here for the question and solution: https://math.meta.stackexchange.com/questions/35669/chrome-doesnt-display-some-symbols

TLDR;

Right-click on the equations in the vignette and change the maths renderer to the option that looks appealing to you.

<img width="956" alt="Screenshot 2024-06-03 at 14 11 22" src="https://github.com/epiforecasts/EpiNow2/assets/43169674/8fed5d4b-edfd-4507-b5b2-40551c414953">
